AboutPage.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <meta charset="UTF-0">
+<title> Renad Alobaid|About Me  </title>
+    <link rel="stylesheet" href="css/style.css">
+    </head>
+<body>
```

Comment:
be sure to indent your code to make it easier to read 

read this link for more information about indentations
https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html